tests/cross_functional/system_test/test_mcg_noobaa_db_ha_and_feature_verification.py

```
@@ -0,0 +1,960 @@
+        object_versions = s3_list_object_versions(
+            mcg_obj,
+            versioning_bucket_post_rebuild.name,
+            prefix=versioning_key,
+        )
+        delete_markers = object_versions.get("DeleteMarkers", [])
+        assert (
+            len(delete_markers) == 1 and delete_markers[0]["IsLatest"]
+        ), (
+            f"Delete marker was not created for latest version of "
+            f"{versioning_bucket_post_rebuild.name}/{versioning_key}"
+        )
```

Ruff (0.15.17)

[warning] Assertion should be broken down into multiple parts

(PT018)